packages/transformed-erc20-extractor/src/index.js

```diff
@@ -0,0 +1,10 @@
+const fetchLogEntries = require('./fetch-log-entries');
+const getEventData = require('./get-event-data');
+
+module.exports = {
+  configure: () => {},
+  eventType: 'TransformedERC20',
+  fetchLogEntries,
+  getEventData,
+  protocolVersion: 3,
```

Comment:
Doesn't really make sense in this context I think since protocols are becoming more modular.

Potentially need to refactor this to a `version` property in the future which actually refers to the event version rather than protocol version. Essentially shorthand for contract address.